### DISCUSSION TITLE
Wrong Flight Level   ?????

### DISCUSSION THREAD

**Discussion by Reisener** [flybywiresim/aircraft#5975] [Help] [answered]
Wrong flight level ????
Hi there,
Somehow the flight level is wrong I suspect.
For example, I fly to FL 340 and QNH values to standard.
On IVAO webeye, however, a completely different height is displayed, always significantly more, like 35581 feet now.
Please can you check that out.
Thank you DLH 874 Bernd
MSFS202 and FBW current DEV version

**Comment by aguther** [comment, marked as answer]
That's depending on the pressure and happens in real life, too. Pressure altitude (which is used for flight levels) is defined by ambient pressure only and that means that depending on the weather (either if you're flying in a zone with high pressure or low pressure) that means that real altitude might be different. But that does not matter because all airplanes will fly on the same "wrong" GPS altitude.

There is also a long discussion ongoing about that issue:
https://forums.flightsimulator.com/t/vatsim-ivao-pilotedge-users-be-aware-of-an-important-bug/426142

![3c14085340a0e5b15b8b209a1f4ae3ec982ce14a](https://user-images.githubusercontent.com/6629914/136623975-251c9bc4-5d2b-4f6f-b8fa-d9639900ed39.jpeg)

**Comment by Reisener** [comment]
Hi there,
Sorry that's all clear to me. I was just amazed that other pilots in the IVAO network wrote to me saying I was on the wrong FL.
We then compared the data of the set FL + STD QNH with other pilots who were within 40nM of my area.
Even so, there were big differences, up to 1500 feet.
However, I forgot to ask which one with which one
SIM is flown --- P3D, FSX MSFS2020 X-Plan.
I just wanted to ask again whether the MSFS2020 or
FBW A320 an error has crept in.
Thanks again for the answer and have a nice day.
DLH874 Bernd

**Reply by tracernz** [reply]
The error is with IVAO. MSFS is the only sim that attempts to model the atmospheric effects to a higher level of fidelity. You can read a good summary from the VATSIM Vpilot developer here: https://forums.flightsimulator.com/t/vatsim-ivao-pilotedge-users-be-aware-of-an-important-bug/426142/363

**Comment by Reisener** [comment]
Hello,
thanks again for the hint. It has been very informative for me, but also very complicated.
Since I only flew P3dv4 before, I didn't know about the problem. Now I wait and see what will happen, but I know that my transmitted altitude is the better and more accurate one.
Thanks again for the effort.
DLH874 Bernd
